composer.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "php/pie",
+    "type": "project",
+    "description": "PIE - the PHP Installer for Extensions",
+    "license": "BSD-3-Clause",
+    "bin": ["bin/pie"],
+    "autoload": {
+        "psr-4": {
+            "Php\\Pie\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Php\\PieUnitTest\\": "test/unit/"
+        }
+    },
+    "authors": [
+        {
+            "name": "James Titcumb",
+            "email": "asgrim@php.net"
+        },
+        {
+            "name": "PIE contributors",
+            "homepage": "https://github.com/php/pie/graphs/contributors"
+        }
+    ],
+    "require": {
+        "php": "8.1.*||8.2.*||8.3.*",
```

Comment:
To be discussed: ThePHPF/pie-design#7

Reply:
Resolution as per https://github.com/ThePHPF/pie-design/issues/7#issuecomment-2003177751

> I think therefore, this discussion can probably be closed. As per PR #1 the tool will initially need PHP 8.1 to be run, but will still be able to install older PHP versions using `--with-php-config`, as mentioned.